english/hammertime/hammertime.md

```diff
@@ -0,0 +1,2 @@
+You break into the hammertime-dance, but quickly realize that this
+doesn't really get to [anywhere](https://github.com/udacity/create-your-own-adventure/blob/master/english/eating-walls/eating-marshmallows.md) 
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.